openlibrary/templates/books/custom_carousel.html

```diff
@@ -43,7 +39,7 @@
           <img class="bookcover" loading="lazy"
             width="130" height="200"
             title="$('%s%s'%(title,byline))"
-            $img_attr="$(cover_url)"/>
+            $img_attr ="$(cover_url)"/>
```

Comment:
Why?

Reply:
I'm unsure of what you mean, is it that I can just replace $img_attr by src directly instead of another variable or that data-lazy should be kept? 
(also will be fixing whitespace(s), they didn't show up yesterday on my editor :/ )

Reply:
I was commenting on that line...  Just the whitespace change.

Thanks for doing this.  I could not find `data-lazy` in this repo or Infogami or web.py so I think you are doing the right thing.